taipy/core/config/job_config.py

```diff
@@ -24,70 +26,68 @@ class JobConfig:
     Parameters:
         mode (str): The Taipy operating mode. By default, the "standalone" mode is set. On Taipy enterprise,
             the "airflow" mode is available.
-        nb_of_workers (int): The maximum number of running workers to execute jobs. It must be a positive integer.
-            The default value is 1.
         **properties: A dictionary of additional properties.
     """
 
     _MODE_KEY = "mode"
     _DEFAULT_MODE = "standalone"
 
-    _NB_OF_WORKERS_KEY = "nb_of_workers"
-    _DEFAULT_NB_OF_WORKERS = 1
+    _MODE_TO_MODULE: Dict[str, str] = {
+        "airflow": "taipy.airflow",
+    }
 
-    def __init__(self, mode: str = None, nb_of_workers: Union[int, str] = None, **properties):
-        self.mode = mode
+    def __init__(self, mode: str = None, **properties):
+        self.mode = mode or self._DEFAULT_MODE
+        self._config_cls = self._get_config_cls(self.mode)
+        self._config = self._create_config(self._config_cls, **properties)
```

Comment:
Following the previous comment, I would do:

```suggestion
        self._config = self._create_config(self._config_cls, nb_of_workers, **properties)
```